### PR TITLE
feat: #454 Added component WayCard

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: mw-webapp/src/component/wayCard/wayTag/WayTag.module.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@import "src/styles/variables";
+
+$WayTagPadding: 4px 10px;
+$WayTagBorderRadius: 1000px;
```
Comment (tritonJS826) [inline mw-webapp/src/component/wayCard/wayTag/WayTag.module.scss:4]: Why 1000px?  If you want to have circled border usually better to use 50%

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/wayCard/wayTag/WayTag.module.scss (hunk 1 above)
Comment (Ekaterina1994) [reply mw-webapp/src/component/wayCard/wayTag/WayTag.module.scss:4]: Value was taken from figma

Comment (Ekaterina1994) [reply mw-webapp/src/component/wayCard/wayTag/WayTag.module.scss:4]: Value 50%  looks different , not like 1000px

**Review by SergioAJS — APPROVED** [review]

File: mw-webapp/src/component/wayCard/wayTag/WayTag.module.scss (hunk 1 above)
Comment (tritonJS826) [reply mw-webapp/src/component/wayCard/wayTag/WayTag.module.scss:4]: Do you think that 1000px better than 50%?

Comment (tritonJS826) [reply mw-webapp/src/component/wayCard/wayTag/WayTag.module.scss:4]: If yes - merge

**Comment by Ekaterina1994** [comment]
Changes was merged in PR #523

**Pull request closed without merging**